code-scanning/properties/credo.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Credo Scan",
+    "creator": "Credo",
```

Comment:
Here the creator is `Credo`. Does `Credo` know about this commitment?

Reply:
@AlexDeMichieli @griffinashe Would you happen to have an update on this? I can reach out to rrrene if not

Reply:
René is aware of this and I just asked him if he'd be open to signing up for our Tech Partner Program. @ElizabethBarrord @Phantsure